extensions/vscode/webviews/homeView/src/components/tree/TreeSection.vue

```diff
@@ -27,6 +27,9 @@
       <div v-if="actions" class="actions">
         <ActionToolbar :title="title" :actions="actions" />
       </div>
+      <div v-if="headerActions" class="header-actions">
+        <ActionToolbar :title="title" :actions="headerActions" />
+      </div>
```

Comment:
I do not think this is the right approach to showing that a section has an error. I would have expected to use the `codicon` prop to pass in `codicon-alert`. The reason for this:
- this is not an action, clicking on it doesn't do anything that the header doesn't already do (expand)
- the icon goes away when expanded removing the glanceability of the icon showing that there is an error

If we do want to go with this method we need to remove the `expanded` prop and instead use the available `v-model` for `expanded`. This will prevent the bug currently present where the button stops working after the initial expansion.

Reply:
I started with a codicon, but that puts it as a left-hand prefix to the label,  and I wanted it at the right side. Reaching out to chat.